### DISCUSSION TITLE
Cascader drop-down width is matching width of Cascader component

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2652] [Q&A] [answered]
In the docs, the drop-down of the Cascader auto sizes to fit the content, but in my project the drop-down is always the full width of the Cascader component. Is this something that has changed? I want to make the Cascader wide so that it can display the path of 3 levels once selected, but the drop-down needs to be much smaller.

**Comment by ElderJames** [comment]
Hi @michaelcsikos , could you please provide a demo that can reproduce this issue?

**Comment by michaelcsikos** [comment, marked as answer]
Hi @ElderJames, please see the following zip:

[CascaderProject.zip](https://github.com/ant-design-blazor/ant-design-blazor/files/9459172/CascaderProject.zip)

![image](https://user-images.githubusercontent.com/21806535/187610391-aa2637de-4362-4793-8d81-a2a66dda0f09.png)

**Reply by ElderJames** [reply]
@michaelcsikos It seems that the AntDesign package for AntDesign.ProLayout was out of date.

**Reply by michaelcsikos** [reply]
@ElderJames  [NuGet](https://www.nuget.org/packages/AntDesign.ProLayout/#versions-body-tab) still has 0.10.6 as the latest version for AntDesign.ProLayout.

**Reply by ElderJames** [reply]
Yes,I will release a new package with the latest AntDesign today, then I will tell you here.

**Reply by ElderJames** [reply]
Hi @michaelcsikos , I have released the latest version for ProLayout, please try again.

**Reply by michaelcsikos** [reply]
Hi @ElderJames I have done a NuGet update and the `Cascader` component doesn't work at all any more. It won't show the drop-down.

**Reply by ElderJames** [reply]
![image](https://user-images.githubusercontent.com/7550366/187842891-a1b19cf2-9748-4880-a1f7-ee9294a9f7dd.png)

**Reply by michaelcsikos** [reply]
Ah, I had to clear my browser cache. Thanks.
